CONTRIBUTING.md

```diff
@@ -24,3 +24,8 @@ documentation generation, and in the search use case, the function signature its
 
 The resultant wrapper functions have signatures that adhere to the FHIR specification, and they
 forward the requests on to the handlers provided by the developer.
+
+# Contributions
+
+If you would like to contribute to fhirstarter, consider discussing your contribution on GitHub, 
```

Comment:
Just to be consistent with the rest of the file

```suggestion
If you would like to contribute to FHIRStarter, consider discussing your contribution on GitHub, 
```

Reply:
In README.md it's consistently FHIRStarter. Should I make it consistent across both files?

Reply:
I checked and except for references in code/URLs (lowercase), I think it's already consistent, except for this new change.